### ISSUE TITLE
cryodrgn parse_ctf_csparc fails when .cs file is missing image size

### ISSUE THREAD

**Issue by zhonge** [ml-struct-bio/cryodrgn#51]
Sometimes the cryosparc CTF metadata file (.cs) doesn't contain the image size. Add additional arguments to `cryodrgn parse_ctf_csparc` to overwrite these fields.

```
$ cryodrgn parse_ctf_csparc J150/P10_J150_passthrough_particles.cs -o ctf.pkl
2021-04-18 13:15:07     1313694 particles
Traceback (most recent call last):
  File "/nobackup/users/zhonge/anaconda3/envs/cryodrgn4/bin/cryodrgn", line 11, in <module>
    load_entry_point('cryodrgn', 'console_scripts', 'cryodrgn')()
  File "/nobackup/users/zhonge/dev/cryodrgn/master/cryodrgn/__main__.py", line 52, in main
    args.func(args)
  File "/nobackup/users/zhonge/dev/cryodrgn/master/cryodrgn/commands/parse_ctf_csparc.py", line 27, in main
    ctf_params[:,0] = metadata['blob/shape'][0][0]
ValueError: no field of name blob/shape
```

**Comment by zhonge** [comment]
Fixed in commit 005afc2563dcefffe17985d562f4d4fe9196129a

```
$ cryodrgn parse_ctf_csparc J150/P10_J150_passthrough_particles.cs -o ctf.pkl -D 128 --Apix 2.49
2021-04-18 13:38:00     1313694 particles
2021-04-18 13:38:00     Image size (pix)  : 128
2021-04-18 13:38:00     A/pix             : 2.49
2021-04-18 13:38:00     DefocusU (A)      : 15666.3037109375
2021-04-18 13:38:00     DefocusV (A)      : 15522.8173828125
2021-04-18 13:38:00     Dfang (deg)       : -85.99214312216307
2021-04-18 13:38:00     voltage (kV)      : 300.0
2021-04-18 13:38:00     cs (mm)           : 2.700000047683716
2021-04-18 13:38:00     w                 : 0.10000000149011612
2021-04-18 13:38:00     Phase shift (deg) : 0.0
2021-04-18 13:38:00     Saving /nobackup/users/zhonge/sandbox/ctf.pkl
```

TODO: update documentation

**Comment by michal-g** [comment]
Documentation has already been updated as necessary, see:
https://ez-lab.gitbook.io/cryodrgn/cryodrgn-empiar-10076-tutorial#id-3.2-convert-ctf-parameters-to-cryodrgn-format
